src/cwt/claim.rs

```diff
@@ -0,0 +1,352 @@
+                OffsetDateTime::new_utc(Date::MAX, Time::from_hms(23, 59, 59).unwrap()),
+                NumericDate::IntegerSeconds(253402300799),
+            ),
+            (
+                "max OffsetDateTime with nanos",
+                OffsetDateTime::new_utc(
+                    Date::MAX,
+                    Time::from_hms_nano(23, 59, 59, 999_999_999).unwrap(),
+                ),
+                NumericDate::FractionalSeconds(253402300799.999999999),
+            ),
+            (
+                "min OffsetDateTime without nanos",
+                OffsetDateTime::new_utc(Date::MIN, Time::from_hms(0, 0, 0).unwrap()),
+                NumericDate::IntegerSeconds(-377705116800),
+            ),
+            (
+                "min OffsetDateTime with 1 nano",
+                OffsetDateTime::new_utc(Date::MIN, Time::from_hms_nano(0, 0, 0, 1).unwrap()),
+                NumericDate::FractionalSeconds(-377705116799.99999999),
+            ),
+            (
+                "UNIX epoch",
+                OffsetDateTime::UNIX_EPOCH,
+                NumericDate::IntegerSeconds(0),
+            ),
+            (
+                "normal date (+) without nanos",
+                OffsetDateTime::new_utc(
+                    Date::from_calendar_date(2024, Month::March, 10).unwrap(),
+                    Time::from_hms(16, 40, 32).unwrap(),
+                ),
+                NumericDate::IntegerSeconds(1710088832),
+            ),
+            (
+                "normal date (-) without",
+                OffsetDateTime::new_utc(
+                    Date::from_calendar_date(1960, Month::June, 24).unwrap(),
+                    Time::from_hms(7, 14, 34).unwrap(),
+                ),
+                NumericDate::IntegerSeconds(-300473126),
+            ),
+            (
+                "normal date (+) with nanos",
+                OffsetDateTime::new_utc(
+                    Date::from_calendar_date(2024, Month::March, 10).unwrap(),
+                    Time::from_hms_nano(16, 40, 32, 235_432_100).unwrap(),
+                ),
+                NumericDate::FractionalSeconds(1710088832.2354321),
+            ),
+            (
+                "normal date (-) with nanos",
+                OffsetDateTime::new_utc(
+                    Date::from_calendar_date(1960, Month::June, 24).unwrap(),
+                    Time::from_hms_nano(7, 14, 34, 800_000_000).unwrap(),
+                ),
+                NumericDate::FractionalSeconds(-300473125.2),
+            ),
+        ]
+    }
+
+    #[test]
+    fn offset_to_numeric_date() {
+        for (case, odt, expected_nd) in success_conversion_cases() {
+            let nd: NumericDate = odt
+                .try_into()
+                .unwrap_or_else(|_| panic!("failed to convert {}", case));
+            assert_eq!(nd, expected_nd, "case: {}", case);
+        }
+    }
+
+    #[test]
+    fn numeric_date_to_offset() {
+        let ignore_cases = ["max OffsetDateTime with nanos"];
```

Comment:
Why is this case ignored?

Reply:
too much floating point error since there are more than 15 sig figs; that said, this is not a final solution for merging! I mostly wanted to flag that this is still the case. Based on the other comment/decision on how to handle this, I can either add custom epsilons per test case (and make this one larger) or else reduce the precision of this case itself.